README.md

```diff
@@ -24,7 +24,7 @@ are requiered sdfdsf
 build system
 
 * Boost (>=1.59)
-* h5cpp (>=0.0.4)
+* [h5cpp](https://github.com/ess-dmsc/h5cpp) (>=0.0.4) -- ATTENTION THIS IS NOT http://h5cpp.org/
```

Comment:
I would change 
`-- ATTENTION THIS IS NOT http://h5cpp.org/` 
to
` -- from https://ess-dmsc.github.io/h5cpp/ (not http://h5cpp.org/)`

Reply:
See #139